tests/resources/serializers/test_dublincore_serializer.py

```diff
@@ -0,0 +1,84 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2021 CERN.
+#
+# Invenio-RDM-Records is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Resources serializers tests."""
+
+from invenio_rdm_records.resources.serializers.dublincore import \
+    DublinCoreJSONSerializer, DublinCoreXMLSerializer
+
```

Comment:
It might be good to add a test for:
- list serializer? If needs implementing - see one of the above comments about it
- In the schema many of the fields would return an empty list `[]` instead of `missing`, for example subjects or descriptions. What is the expected output for the `minimal_record`. I.e. should the serialized field be an empty array or not be present?. Adding a test for it might be a good idea.

Reply:
added test cases for minimal records and list serializer